uber/models.py

```diff
@@ -474,8 +474,8 @@ def jobs_for_signups(self):
 
         def guess_attendee_watchentry(self, attendee):
             return self.query(WatchList).filter(and_(or_(WatchList.first_names.contains(attendee.first_name),
-                                                         WatchList.email == attendee.email,
-                                                         WatchList.birthdate == attendee.birthdate),
+                                                         and_(WatchList.email != '',WatchList.email == attendee.email),
```

Comment:
I also forgot to mention that you can run the command ``pep8`` on your Vagrant machine.  You pass the directory you want to check.  For example,

```bash
(env)vagrant@vagrant-ubuntu-trusty-64:~$ pep8 uber/sideboard/plugins/uber/uber
```